### PR TITLE
Add Jambalaya members to homepage

### PR DESCRIPTION
**Pull request by RCdeWit**

### CODE REVIEW

**Comment by jellebouwman** [comment]
@RCdeWit how does this look in Storybook? 📖

**Review by jellebouwman — APPROVED** [review]
LGTM! Thanks for making a PR!

File: index.html [hunk 1]
```diff
@@ -11,6 +11,14 @@
 
     <h1><span class="heading-bg">Hello world!</span></h1>
 
+    <p>We are Jambalaya:</p>
+    <ul>
+      <li>Bruno Lauwers - <a href="https://brunolauwers.nl/">Infinite8</a></li>
+      <li>Jelle Bouwman - <a href="https://yousable.nl/">Yousable</a></li>
+      <li>Laurens Duijvesteijn - <a href="https://duijf.io">Immutative</a></li>
```
Comment (duijf) [inline index.html]:
```suggestion
      <li><a href="https://duijf.io/">Laurens Duijvesteijn</a></li>
```

**Review by duijf — APPROVED** [review]
Thanks!

**Pull request merged**